### DISCUSSION TITLE
モバイル学習コンテンツで、ModalはAndroidでは動作が変わらない旨を記載したい

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#260] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/basic-concepts/react-navigation-basics/modal
Modal

実際に画面を動かしてもモーダルになっているのかわからない。
ヘッダーがないだけ？
→以下にAndroidでは動作が変わらない説明が記載されていた。
　なので記載場所をアプリの実装前に/React Navigationの基本/Modalにしても良いかも。

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/screens/modal
Modalスクリーン

**Comment by nagano-ryo** [comment]
pull request
https://github.com/ws-4020/mobile-app-crib-notes/pull/277

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
